_posts/en/newsletters/2020-10-14-newsletter.md

```diff
@@ -0,0 +1,175 @@
+summarizes discussion about LN upfront payments, describes a mailing
+list thread about updating bech32 addresses for taproot, and links to an
+updated proposal for an alternative way to secure LN payments.  Also
+included are our regular sections with summaries of a Bitcoin Core PR
+Review Club meeting, releases and release candidates, and notable
+changes to popular Bitcoin infrastructure software.
+
+## Action items
+
+- **Upgrade LND to 0.11.x:** the LND development team posted
+  [announcements][lnd warning] to the Lightning-Dev and LND Engineering
+  mailing lists warning users about the planned disclosure on 20 October
+  2020 of vulnerabilities affecting LND versions 0.10.x and earlier.
+  The team "strongly urge[s] the community to upgrade to lnd 0.11.0 or
+  above [as soon as possible]."  (Note: the mailing list archive
+  software slightly alters the text of the announcement, so anyone
+  wanting to verify the PGP signature on the announcement should follow
+  some [additional steps][lnd pgp].)
+
+## News
+
+- **LN upfront payments:** in the current LN protocol, the attempt to
+  route a payment can lock part of a routing node's funds for hours or
+  days.  Since routing fees are only paid if a payment succeeds, an
+  attacker can use payments designed to fail in order to prevent a nodes
+  from earning income on the capital they store in their channels.  One
+  previously discussed option that may better align incentives is to
+  charge a non-refundable fee upfront when a routing request is received
+  (see [Newsletter #72][news72 upfront payments]).
+
+    This week on the Lightning-Dev mailing list, a
+    [discussion][teinturier dynamic] about a proposed minor protocol
+    change transformed into a conversation about upfront fees:
+
+    - *Incremental routing:* ZmnSCPxj [described][zmnscpxj tunneling]
+      a nested encrypted tunneling protocol where routes would be
+      built incrementally, with the spender being able to pay each
+      successive routing hop independently.  This would ensure routing
+      fees couldn't be stolen by an earlier hop that deliberately failed
+      a route.  A downside to this approach is that it would require a
+      significant number of network round trips, which could make even
+      successful payments take a long time.  A spy node who kept track
+      of routed message timing and HTLC duration could also build rough
+      estimates about how many hops away the spender or receiver are
+      from it, reducing the amount of privacy users would receive from
+      LN.
+
+    - *Trusted upfront payment:* Antoine Riard [promoted][riard trust]
+      the idea of simply paying the upfront fees and, if your peer
+      steals them, downgrade their score so that your routing
+      algorithm prefers not to use them again.  Assuming a single
+      upfront fee is much smaller than the aggregate amount of fees a
+      peer could earn over weeks or months of routing, there should be
```

Comment:
maybe s/should/would/

Reply:
I prefer *should* as there could be a reason the incentive doesn't apply.